sssd_test_framework/roles/keycloak.py

```diff
@@ -126,12 +129,66 @@ def add(
 
         return self
 
-    def delete(self) -> None:
+    def delete(
+        self,
+    ) -> SSHProcessResult:
         """
         Delete Keycloak user.
+
+        :return: SSH exec result
+        :rtype: SSHProcessResult
         """
         del_user = f"delete users/{self.id}"
-        self.role.kcadm(del_user)
+        result = self.role.kcadm(del_user)
+        return result
+
+    def modify(
+        self,
+        *,
+        firstName: str | None = None,
+        lastName: str | None = None,
+        email: str | None = None,
+        enabled: bool | None = None,
+    ) -> KeycloakUser:
+        """
+        Modify keycloak user attributes.
+
+        Parameters that are not set are ignored.
+
+        :param firstName: User's first name
+        :type firstName: str | None, optional
+        :param lastName: User's last name
+        :type lastName: str | None, optional
+        :param email: User's email address
+        :type email: str | None, optional
+        :param enabled: User account enabled/disabled status
+        :type enabled: bool | None, optional
+        :return: SSH exec result
+        :rtype: SSHProcessResult
+        """
+
```

Comment:
No empty line after the comments, sorry to be nitpicky but just following what most of our code looks like.

Reply:
removed